docs/getting-started/edfi-exchange/ods-5-3-deployment-on-azure.md

```diff
@@ -0,0 +1,29 @@
+---
+owned_by: Stephen Fuqua
+last_updated: 12/10/2024 Robert Hunter Jr
```

Comment:
Do we need these tags? I certainly don't need to own this.

Reply:
Sure don't. This was an artifact from my testing FrontMatter.